src/main/java/org/opentripplanner/routing/trippattern/TripTimes.java

```diff
@@ -86,6 +86,11 @@
      */
     int[] departureTimes;
 
+    /**
+     * Keep track of stop time objects to enable notices to point to a specific stop time.
+     */
+    StopTime[] stopTimes;
+
```

Comment:
An alternative would be to use a multimap in the index, but the code gets easier to understand this way.